src/ReturnBookControl.java

```diff
@@ -1,33 +1,42 @@
 public class ReturnBookControl {
 
-    private ReturnBookUI ui;
-    private enum CONTROL_STATE {INITIALISED, READY, INSPECTING};
-    private CONTROL_STATE state;
+    private ReturnBookUi ui;
+    
+    private enum ControlState { INITIALISED, READY, INSPECTING };
+    
+    private ControlState state;
+    
     private Library library;
+    
     private Loan currentLoan;
 
     
     public ReturnBookControl() {
         this.library = Library.INSTANCE();
-        state = CONTROL_STATE.INITIALISED;
+        
+        state = ControlState.INITIALISED;
     }
 
     
-    public void setUI(ReturnBookUI ui) {
-        if (!state.equals(CONTROL_STATE.INITIALISED)) {
+    public void setUi(ReturnBookUi ui) {
+        if (!state.equals(ControlState.INITIALISED)) {
             throw new RuntimeException("ReturnBookControl: cannot call setUI except in INITIALISED state");
         }
```

Comment:
Add a new line in between lines 24 and 25 to make spacing uniform throughout file.